src/main/java/athena/logic/Parser.java

```diff
@@ -94,9 +94,8 @@ public static Command parseAddCommand(String taskInfo, int namePos, int timePos,
             throws InvalidCommandException {
         String nullDefault = "";
```

Comment:
I think `nullDefault` as a variable name isn't really clear. Maybe `emptyString`? Also it might be better to have this as a constant define at the top of the class. So maybe `EMPTY_STRING`?